lib/jsdom/living/events/TouchEvent-impl.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
+
+const UIEventImpl = require("./UIEvent-impl").implementation;
+
+class TouchEventImpl extends UIEventImpl {
```

Comment:
A bit sad that all of this could be reduced to `exports.implementation = class extends UIEventImpl {}` but I guess I am OK with the verboseness for the sake of uniformity...